.travis.yml

```diff
@@ -1,28 +1,40 @@
 language: python
-script:
-  - ./_travis/run.sh
+sudo: false
+
 before_install:
   - env
   - openssl version
   - python -c "import ssl; print(ssl.OPENSSL_VERSION)"
+
+install:
+  - ./_travis/install.sh
+
+script:
+  - ./_travis/run.sh
+
 after_success:
   - ./_travis/upload_coverage.sh
+
 cache:
   directories:
   - ${HOME}/.cache
-install:
-  - ./_travis/install.sh
+
 notifications:
   email: false
+
 env:
   global:
     - GAE_SDK_PATH=${HOME}/.cache/google_appengine
     - PYTHONWARNINGS=always::DeprecationWarning
+
+    - PYPI_USERNAME=shazow
```

Comment:
Let's start a new urllib3 account, yeah? I can take care of that. This can stay as-is for this PR, though.

https://github.com/urllib3/urllib3/issues/1522